tests/e2e/README.md

```diff
@@ -9,6 +9,16 @@ Developers, on the other hand, are recommended to run the tests locally before s
 
 # Running E2E Tests
 
+Image registry option:
+
+1. [In-cluster registry](/local/localregistry) 
+If environment variable `HUB` is not set, local registry will be setup and used automatically.
+*MacOS ONLY*
+Add `docker.for.mac.localhost:5000` to insecure registries in the docker daemon setting before running the tests.
+
+2. Remote registry
+By setting `HUB` to your remote registry, the tests will push and pull images from the remote.
```

Comment:
Just to be slightly more explicit with the language:

If the environment variables `HUB` and `TAG` are set, all test deployments will pull images from the specified  docker registry.